### PR TITLE
LINK-2026 | Don't require organization membership from superusers

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Superusers are no longer required to be organization admins / members to perform actions like creating or updating an organization.
### Closes
[LINK-2026](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2026)

[LINK-2026]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr911.api.dev.hel.ninja** :rocket::rocket::rocket:

File: events/permissions.py [hunk 1]
```diff
@@ -69,7 +72,8 @@ class IsObjectEditableByUser(BasePermission):
     def has_object_permission(self, request, view, obj):
         if request.method in permissions.SAFE_METHODS:
             return True
-        return obj.can_be_edited_by(request.user)
+
+        return request.user.is_superuser or obj.can_be_edited_by(request.user)
```
[the remark this answers is not in the document]
Comment (harriris-vincit) [reply events/permissions.py]: Now changed to rely on "can_be_edited_by" only.

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr911.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr911.api.dev.hel.ninja** :rocket::rocket::rocket:

File: linkedevents/serializers.py [hunk 2]
```diff
@@ -303,50 +303,64 @@ def validate_id(self, value):
...
-                        "must be left blank or set to %(required)s or any other organization "
-                        "the user belongs to."
-                    )
-                    % {
-                        "field": str(field),
-                        "given": str(value),
-                        "required": str(publisher),
-                    }
+        if value not in allowed_organizations:
+            publisher = self.context["publisher"]
```
Comment (nicobav) [inline linkedevents/serializers.py:330]: this can lead to potential `KeyError`. 
```suggestion
            publisher = self.context.get("publisher")
```

**Review by nicobav — APPROVED** [review]

**Pull request merged**